### DISCUSSION TITLE
Size behavior in grid inputs

### DISCUSSION THREAD

**Discussion by emiCacc** [primefaces/community#887] [PrimeNG]
Hello.
I have a doubt with the behavior of the inputs of my table. When I click inside it to enter a value, it enlarges, which is fine, but it increases the size of the column dynamically. According to the documentation of prime Ng, this is the expected behavior, however I would like to modify this measure, is there any way to manipulate this class.
Thanks and greetings.

![vid1](https://github.com/primefaces/community/assets/11484411/3b0aa409-233b-4f82-86ca-dd432f3201b4)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz reproducer so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
